addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -470,11 +523,30 @@ def _import_fattura_pa(self, tree, invoice):
                                 invoice_line_form.quantity = 1
 
                             # Taxes
-                            tax_element = element.xpath('.//AliquotaIVA')
+                            percentage = None
+                            unit_price = 0
+                            if not simplified:
+                                tax_element = element.xpath('.//AliquotaIVA')
+                                if tax_element and tax_element[0].text:
+                                    percentage = float(tax_element[0].text)
+                            else:
+                                amount_element = element.xpath('.//Importo')
+                                if amount_element and amount_element[0].text:
+                                    amount = float(amount_element[0].text)
+                                    tax_element = element.xpath('.//Aliquota')
+                                    if tax_element and tax_element[0].text:
+                                        percentage = float(tax_element[0].text)
+                                        unit_price = amount / (1 + percentage / 100)
+                                    else:
+                                        tax_element = element.xpath('.//Imposta')
+                                        if tax_element and tax_element[0].text:
+                                            tax_amount = float(tax_element[0].text)
+                                            unit_price = amount - tax_amount
+                                            percentage = tax_amount/unit_price * 100
```

Comment:
```suggestion
                                            percentage = tax_amount / unit_price * 100
```

Reply:
maybe some rounding needed here as well...  (how did this go in the tests?)

Reply:
rounding is indeed needed